src/CONST.ts

```diff
@@ -5459,6 +5464,14 @@ const CONST = {
                 description: 'workspace.upgrade.taxCodes.description' as const,
                 icon: 'Coins',
             },
+            rules: {
+                id: 'rules' as const,
+                alias: 'rules',
+                name: 'Rules',
+                title: 'workspace.upgrade.rules.title' as const,
+                description: 'workspace.upgrade.rules.description' as const,
```

Comment:
1. Do we need as const in those fields?
2. If yes, why wouldn't we use as const for the whole array/object

Reply:
It has been implemented according to the other consts in this getter